docs/sources/reference/components/database_observability/database_observability.mysql.md

```diff
@@ -0,0 +1,90 @@
+---
+canonical: https://grafana.com/docs/alloy/latest/reference/components/database_observability.mysql/
+description: Learn about database_observability.mysql
+title: database_observability.mysql
+---
+
+<span class="badge docs-labels__stage docs-labels__item">Experimental</span>
+
+# database_observability.mysql
+
+{{< docs/shared lookup="stability/experimental.md" source="alloy" version="<ALLOY_VERSION>" >}}
+
+## Usage
+
+```alloy
+database_observability.mysql "LABEL" {
+  data_source_name = DATA_SOURCE_NAME
+  forward_to       = [LOKI_RECEIVERS]
+}
+```
+
+## Arguments
+
+The following arguments are supported:
+
+| Name                 | Type           | Description                                                                                                         | Default | Required |
+| -------------------- | -------------- | ------------------------------------------------------------------------------------------------------------------- | ------- | -------- |
+| `data_source_name`   | `secret`       | [Data Source Name](https://github.com/go-sql-driver/mysql#dsn-data-source-name) for the MySQL server to connect to. |         | yes      |
+
+## Blocks
+
+The following blocks are supported inside the definition of `database_observability.mysql`:
+
+| Hierarchy | Block | Description | Required |
+| --------- | ----- | ----------- | -------- |
+|           |       |             |          |
```

Comment:
nit: Replace this area by "The `database_observability.mysql` component does not support any blocks, and is configured fully through arguments."

Reply:
Fixed